lib/reference/keywordSets.js

```diff
@@ -677,6 +677,7 @@ keywordSets.mediaFeatureNames = uniteSets(keywordSets.deprecatedMediaFeatureName
 	'color-gamut',
 	'color-index',
 	'dynamic-range',
+	'display-mode',
```

Comment:
Sort in alphabetical order, please?

```suggestion
	'display-mode',
	'dynamic-range',
```

Reply:
@ybiquitous You are welcome! Thanks for the suggestion.